pilot/docker/Dockerfile.proxyv2

```diff
@@ -47,8 +48,7 @@ COPY pilot-agent /usr/local/bin/pilot-agent
 
 COPY envoy_policy.yaml.tmpl /var/lib/istio/envoy/envoy_policy.yaml.tmpl
 
-COPY stats-filter.wasm /etc/istio/extensions/stats-filter.wasm
-COPY metadata-exchange-filter.wasm /etc/istio/extensions/metadata-exchange-filter.wasm
+COPY . *.wasm /etc/istio/extensions/
```

Comment:
isn't this copying `.` which is everything?

Reply:
MOSN does not have two files, stats-filter.wasm and metadata-exchange-filter.wasm. COPY. *.Wasm /etc/istio/extensions/ is to prevent Dockerfile from reporting errors when there are no two files

Reply:
You are copying the entire docker context to the extensions folder though rather than nothing, I think.

Reply:
problem solved.